test/bats/helpers/common-setup.bash

```diff
@@ -19,8 +19,8 @@ _common_setup() {
 
     # Kurtosis enclave and service identifiers
     readonly enclave=${KURTOSIS_ENCLAVE:-aggkit}
-    readonly contracts_container=${KURTOSIS_CONTRACTS:-contracts-001}
-    readonly contracts_service_wrapper=${KURTOSIS_CONTRACTS_WRAPPER:-"kurtosis service exec $enclave $contracts_container"}
-    readonly erigon_rpc_node=${KURTOSIS_ERIGON_RPC:-cdk-erigon-rpc-001}
-    readonly l2_rpc_url=${L2_ETH_RPC_URL:-"$(kurtosis port print $enclave $erigon_rpc_node rpc)"}
+    # readonly contracts_container=${KURTOSIS_CONTRACTS:-contracts-001}
+    # readonly contracts_service_wrapper=${KURTOSIS_CONTRACTS_WRAPPER:-"kurtosis service exec $enclave $contracts_container"}
+    # readonly erigon_rpc_node=${KURTOSIS_ERIGON_RPC:-cdk-erigon-rpc-001}
+    # readonly l2_rpc_url=${L2_ETH_RPC_URL:-"$(kurtosis port print $enclave $erigon_rpc_node rpc)"}
```

Comment:
Why is this commented out? If I'm not mistaken, we are using these vars in the tests (and it could be the reason why the tests are failing atm).